voc/python/ast.py

```diff
@@ -221,10 +224,17 @@ def convert_to_Name(_node):
                 lineno=_node.lineno,
                 col_offset=_node.col_offset
             )
+
+        # The node is a regular yield statement
+        # Mark it as 'regular_yield' to flush the generator's message upon restored
+        if isinstance(node, ast.Yield):
+            setattr(node, "regular_yield", True)
```

Comment:
What's the alternative to "regular" yield?

Reply:
Would `is_yield_stament` be better?

Reply:
It's not (necessarily) a question of naming - I don't understand what a "regular yield" *is*. What's an "irregular" yield? How do you get an "irregular" yield?

Reply:
Initially I want to differentiate a yield statement (e.g. the plain old `yield 123`, which I called "regular yield") from the yield expression (`x = yield 123`).

However, I did some code refactoring just now and the attribute is no longer needed.